src/python/WMCore/Storage/StageOutImpl.py

```diff
@@ -143,6 +143,16 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
         """
         raise NotImplementedError("StageOutImpl.createStageOutCommand")
 
+    def createDebuggingCommand(self, sourcePFN, targetPFN, options=None, checksums=None):
+        """
+        _createDebuggingCommand_
```

Comment:
I think this style used to be used in the old docstring style. For more up-to-date recommendations, please refer to:
https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices